Sources/MarathonCore/ScriptManager.swift

```diff
@@ -321,4 +352,14 @@ internal final class ScriptManager {
             return path
         }
     }
+    
+    private func makeLinuxMainTestFileContent(for script: Script) -> String {
+        let content = "import XCTest\n" +
```

Comment:
How about just returning the expression directly instead of having this temporary variable?